src/core/tiledscene/qgstiledscenelayerrenderer.cpp

```diff
@@ -297,6 +307,7 @@ void QgsTiledSceneLayerRenderer::renderTile( const QgsTiledSceneTile &tile, QgsT
         TileDetails details;
         details.boundary = volumePolygon;
         details.hasContent = hasContent;
+        details.id = QString::number( tile.id() );
```

Comment:
Just a thought: given that these IDs are not stable, and not coming from the underlying dataset, maybe it may be useful to have also some kind of "stable" string IDs for display purposes, that would represent path to the tile? (e.g. one extra character for each level)

Reply:
that's a good idea! I was originally thinking the id can be used along with a `layer.dataProvider().index().getTile( 123 )` command, so I don't think it's strictly necessary to have the stable ids (even though it IS desirable)

Reply:
right!